guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc

```diff
@@ -121,6 +121,8 @@ include::proc_creating-a-host-group-for-each-lifecycle-environment.adoc[leveloff
 
 Use this procedure to change the group of a host.
 
+If you reprovision a host after changing the group, the settings inherited from the host group will be overwritten.
```

Comment:
@melcorr I'm wondering that this note might be better in the intro before the procedure, because inside the step it kinda breaks the flow of the procedure. What do you think?

Reply:
`the settings` in here may be misleading a bit, since Settings is something user can set as really the default value. Such value can be overwritten on host group level and host level. So unless we want to explain the whole inheritance chain here, I'd rather avoid that word.